ws/client/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "jsonrpc-ws-client"
+version = "0.1.0"
+authors = ["David Craven <david@craven.ch>"]
+edition = "2018"
+
+[dependencies]
+failure = "0.1"
+futures = "0.1"
+jsonrpc-core = { version = "*", path = "../../core" }
```

Comment:
Please specify a verison here, otherwise we won;'t be able to push it to crates.io